scripts/us_census/pep/population_estimate_by_race/states_to_shortform.py

```diff
@@ -0,0 +1,61 @@
+_STATES = {
```

Comment:
ditto, please use: https://github.com/datacommonsorg/data/blob/master/util/name_to_alpha2.py

also, use _ prefix for module private vars. in this case, it's an exported constant so should not have _

Reply:
Hi, this part has been moved as a part of another PR. PEP by Race and Monthly Population has been seperated.